src/models/index.ts

```diff
@@ -1 +1,3 @@
 export { default as User } from './user';
+export { default as Group } from './group';
+export { default as UserGroup } from './user-group';
```

Comment:
it's better to use names that start with the small letter (like `userGroup`, not `UserGroup`) because you have here instance. not the class itself. It's not significant but may be confusing.

Reply:
Oh, here it's correct. I haven't noticed that it's db init. Don't mind the comment above :)